src/org/ods/orchestration/phases/FinalizeOdsComponent.groovy

```diff
@@ -30,7 +29,7 @@ class FinalizeOdsComponent {
         this.logger = logger
     }
 
-    public void run(Map repo, String baseDir) {
+    void run(Map repo) {
```

Comment:
NOTE: This method is basically unchanged. The huge diff is only due to the removal of `steps.dir(baseDir) {}`. Switching to the correct working dir is done by the surrounding code now to reduce code duplication.